janus_server/src/message.rs

```diff
@@ -208,19 +213,31 @@ impl Display for Duration {
 
 /// PPM protocol message representing an instant in time with a resolution of seconds.
 #[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, PartialOrd, Ord)]
-pub struct Time(pub(crate) u64);
+pub struct Time(u64);
 
 impl Time {
-    pub(crate) fn as_naive_date_time(&self) -> NaiveDateTime {
+    /// Convert this [`Time`] into a [`NaiveDateTime`], representing an instant in the UTC timezone.
+    pub fn as_naive_date_time(&self) -> NaiveDateTime {
         NaiveDateTime::from_timestamp(self.0 as i64, 0)
     }
 
+    /// Convert a [`NaiveDateTime`] representing an instant in the UTC timezone into a [`Time`].
     pub fn from_naive_date_time(time: NaiveDateTime) -> Self {
         Self(time.timestamp() as u64)
     }
 
+    /// Get the Unix timestamp represented by this [`Time`] (in seconds).
+    pub fn as_timestamp(&self) -> u64 {
+        self.0
+    }
+
+    /// Construct a [`Time`] from a Unix timestamp (in seconds).
+    pub fn from_timestamp(timestamp: u64) -> Self {
+        Self(timestamp)
```

Comment:
I think `timestamp` is too ambiguous. At the risk of being verbose, how about `fn as_seconds_since_epoch()` and `from_seconds_since_epoch()`?